client/src/features/project/components/projectMigration/ProjectEntityMigration.tsx

```diff
@@ -0,0 +1,391 @@
+            Migrate this project to Renku 2.0
+          </Button>
+          <Link
+            to={Links.RENKU_2_MIGRATION_INFO}
+            className={cx("btn", "btn-outline-warning")}
+            rel="noreferrer noopener"
+            target="_blank"
+          >
+            Learn more
+          </Link>
+        </div>
+      </WarnAlert>
+      <MigrationModal
+        isOpen={isOpenModal}
+        toggle={toggle}
+        description={description?.isLoading ? undefined : description?.value}
+        tagList={tagList}
+        projectMetadata={projectMetadata as ProjectMetadata}
+      />
+    </>
+  );
+}
+
+function MigrationModal({
+  isOpen,
+  toggle,
+  projectMetadata,
+  description,
+  tagList,
+}: {
+  isOpen: boolean;
+  toggle: () => void;
+  projectMetadata: ProjectMetadata;
+  description?: string;
+  tagList: string[];
+}) {
+  const {
+    control,
+    formState: { dirtyFields, errors },
+    handleSubmit,
+    watch,
+    setValue,
+  } = useForm<ProjectMigrationForm>({
+    defaultValues: {
+      name: projectMetadata.title,
+      namespace: "",
+      slug: projectMetadata.path,
+      visibility:
+        projectMetadata.visibility === "public" ? "public" : "private",
+    },
+  });
+  const [migrateProject, result] =
+    usePostRenkuV1ProjectsByV1IdMigrationsMutation();
+  const {
+    registryTag,
+    isFetchingData,
+    projectConfig,
+    branch,
+    commits,
+    templateName,
+    resourcePools,
+    isProjectSupported,
+  } = useGetSessionLauncherData();
+
+  const isPinnedImage = !!projectConfig?.config?.sessions?.dockerImage;
+
+  const containerImage = useMemo(() => {
+    return (
+      projectConfig?.config?.sessions?.dockerImage ?? registryTag?.location
+    );
+  }, [projectConfig, registryTag]);
+
+  const defaultSessionClass = useMemo(
+    () =>
+      resourcePools?.flatMap((pool) => pool.classes).find((c) => c.default) ??
+      resourcePools?.find(() => true)?.classes[0] ??
+      undefined,
+    [resourcePools]
+  );
+
+  const resourceClass = useMemo(() => {
+    return (
+      resourcePools
+        ?.flatMap((pool) => pool.classes)
+        .find((c) => c.id == defaultSessionClass?.id && c.matching) ??
+      resourcePools
+        ?.filter((pool) => pool.default)
+        .flatMap((pool) => pool.classes)
+        .find((c) => c.matching) ??
+      resourcePools
+        ?.flatMap((pool) => pool.classes)
+        .find((c) => c.id == defaultSessionClass?.id) ??
+      undefined
+    );
+  }, [resourcePools, defaultSessionClass]);
+
+  const linkToProject = useMemo(() => {
+    return result?.data
+      ? generatePath(ABSOLUTE_ROUTES.v2.projects.show.root, {
+          namespace: result.data.namespace,
+          slug: result.data.slug,
+        })
+      : "";
+  }, [result.data]);
+
+  const onSubmit = useCallback(
+    (data: ProjectMigrationForm) => {
+      if (!containerImage) return;
+      const nowFormatted = DateTime.now().toFormat("yyyy-MM-dd HH:mm:ss");
```

Comment:
Suggestion:
```suggestion
      const nowFormatted = toHumanDateTime({ datetime: DateTime.now() });
```
This will ensure that the datetime format is consistent.